doc/dynamo0p3.rst

```diff
@@ -61,6 +61,14 @@ Operator
 
```

Comment:
At the start of the Kernel section I think it should be explained that there are different types of Kernel with their own rules and that these will be explained in the following sections.